<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1837</width>
    <height>1267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Renderer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_graphicsView">
      <item>
       <widget class="QGraphicsView" name="graphicsView_image">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(121, 134, 255);</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
        </property>
        <property name="renderHints">
         <set>QPainter::RenderHint::Antialiasing|QPainter::RenderHint::LosslessImageRendering|QPainter::RenderHint::TextAntialiasing</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_graphicsView_labels">
        <item>
         <widget class="QLabel" name="label_timer">
          <property name="styleSheet">
           <string notr="true">color: rgb(56, 66, 255);</string>
          </property>
          <property name="text">
           <string>0 ms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fps">
          <property name="styleSheet">
           <string notr="true">color: rgb(56, 66, 255);</string>
          </property>
          <property name="text">
           <string>0 FPS</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_imgColorType">
          <property name="styleSheet">
           <string notr="true">color: rgb(56, 66, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="controlpanel" native="true">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="textinp_filename">
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="accept_filename">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(151, 174, 255);</string>
         </property>
         <property name="text">
          <string>Load file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_transform">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <attribute name="title">
           <string>Transform</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Crop</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_crop">
             <item>
              <widget class="QSpinBox" name="spinBox_crop_left">
               <property name="prefix">
                <string>L </string>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_crop_right">
               <property name="prefix">
                <string>R </string>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_crop_top">
               <property name="prefix">
                <string>T </string>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_crop_bottom">
               <property name="prefix">
                <string>B </string>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Rotate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_rotate">
             <item>
              <property name="text">
               <string>0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>90</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>180</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>270</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_filters">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_interactiveRender">
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
         <property name="text">
          <string>Interactive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_edit">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(151, 174, 255);</string>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_log">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2_log">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>338</width>
            <height>518</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_log">
             <property name="styleSheet">
              <string notr="true">color: rgb(86, 80, 255);
background-color: rgb(155, 255, 158);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::TextFormat::PlainText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_save">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(151, 174, 255);</string>
         </property>
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1837</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
